src/Mount.cpp

```diff
@@ -107,19 +67,17 @@ const float siderealDegreesInHour = 14.95904348958;
 // CTOR
 //
 /////////////////////////////////
-Mount::Mount(float stepsPerRADegree, float stepsPerDECDegree, LcdMenu* lcdMenu) {
-   _instance = this;
-   
-  #if RA_DRIVER_TYPE != DRIVER_TYPE_ULN2003
-  _stepsPerRADegree = stepsPerRADegree * SET_MICROSTEPPING; // hier
-  #else
-  _stepsPerRADegree = stepsPerRADegree;
-  #endif
-  #if DEC_DRIVER_TYPE != DRIVER_TYPE_ULN2003
-  _stepsPerDECDegree = stepsPerDECDegree * DEC_SLEW_MICROSTEPPING;
-  #else
-  _stepsPerDECDegree = stepsPerDECDegree;
+Mount::Mount(LcdMenu* lcdMenu) :
+  _stepsPerRADegree(RA_STEPS_PER_DEGREE),   // u-steps per degree when slewing
+  _stepsPerDECDegree(DEC_STEPS_PER_DEGREE)  // u-steps per degree when slewing
+  #if AZIMUTH_ALTITUDE_MOTORS == 1
+    , _stepsPerAZDegree(AZIMUTH_STEPS_PER_REV / 360),
```

Comment:
we are using 2 different ways to calculate steps per deg. for RA and DEC it gets calculated completelly in adv configs, for alt and az it gets divided by 360 here ... consistency would make it cleaner

Reply:
Agreed. I haven't reviewed the AZ/ALT implementation yet.

Reply:
This can be done in another PR, probably with Camerons input...